modules/parsers/parser-hana/src/main/java/models/TableFunctionDefinitionModel.java

```diff
@@ -13,31 +13,20 @@
 
 import java.util.Objects;
 
-public class TableFunctionDefinitionModel {
+public class TableFunctionDefinitionModel extends DefinitionModel {
 
-  private final String schema;
-  private final String name;
 
-  public TableFunctionDefinitionModel(String schema, String name) {
-    this.schema = schema;
-    this.name = name;
-  }
-
-  public String getSchema() {
-    return schema;
-  }
-
-  public String getName() {
-    return name;
-  }
+    public TableFunctionDefinitionModel(String schema, String name) {
+        super(schema, name);
+    }
 
-  public void checkForAllMandatoryFieldsPresence() {
-    checkPresence(name, "name");
-  }
+    public void checkForAllMandatoryFieldsPresence() {
+        checkPresence(this.getName(), "name");
+    }
 
-  private <T> void checkPresence(T field, String fieldName) {
-    if (Objects.isNull(field)) {
-      throw new TableFunctionMissingPropertyException("Missing mandatory field " + fieldName);
+    private <T> void checkPresence(T field, String fieldName) {
+        if (Objects.isNull(field)) {
```

Comment:
Why not use `field == null`? `Objects.isNull` is useful only for functional stuff being used as a predicate